.markdownlint.jsonc

```diff
@@ -7,5 +7,6 @@
   "MD040": false, // fenced-code-language Fenced code blocks should have a language specified
   "MD033": false, // no-inline-html Inline HTML
   "MD049": { "style": "underscore" }, // enforce use of underscore for em/italics
-  "MD050": { "style": "asterisk" } // enforce use of asterisk for strong/bold
+  "MD050": { "style": "asterisk" }, // enforce use of asterisk for strong/bold
+  "MD051": false
```

Comment:
@dichotommy, why are we adding this? Shouldn't we check for links that don't correspond to a heading?

Reply:
This rule causes errors on all "link fragments", basically any link that starts with `#`. If we enable this rule, we will not be able to quickly write a link to a specific section of the same page using `#` anymore; we will have to write out the full filepath.

Example: instead of `[this is a link](#actions)`, we would have to write `[this is a link](/reference/api/keys.md#actions)`.

It's not a big difference. I chose to disable the rule only because it was the easiest way to get the checks working again—that, and as a lazy person I enjoy the efficiency of link fragments.

Another option besides disabling this rule would be to update the link fragments within this PR (or merge this one, then open a new one that both enables the rule and updates the link fragments to use the filepath).